eng/Packages.Data.props

```diff
@@ -102,7 +102,7 @@
     <PackageReference Update="System.Text.Json" Version="4.7.2" />
     <PackageReference Update="System.Text.Encodings.Web" Version="4.7.2" />
     <PackageReference Update="System.ValueTuple" Version="4.5.0" />
-    <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
+    <PackageReference Update="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
```

Comment:
```suggestion
```

I don't think we want to make this here; this is part of your other PR to bump STJ at the same time.  Let's keep all of these bumps in one place so that they can be validated/merged together.

Reply:
You're doing a direct override for v6 for this specific library.   The package file change has no impact here.   These should ALL be in the main PR.    If necessary, add temporary project references to Azure.Core in the packages that need them.